tests/x509/test_x509_ext.py

```diff
@@ -1662,33 +1662,36 @@ def test_key_cert_sign_crl_sign(self, backend):
 
 class TestDNSName(object):
     def test_init(self):
-        with pytest.warns(utils.DeprecatedIn21):
-            name = x509.DNSName(u"*.\xf5\xe4\xf6\xfc.example.com")
-        assert name.bytes_value == b"*.xn--4ca7aey.example.com"
+        name = x509.DNSName(u".xn--4ca7aey.example.com")
```

Comment:
Why did you drop the `*` fromt his test?

Reply:
That'd be a mistake.